Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/RiskLevelChangeDetector.kt

```diff
@@ -47,6 +48,13 @@ class RiskLevelChangeDetector @Inject constructor(
         val oldResult = changedLevels.first()
         val newResult = changedLevels.last()
 
+        val lastCheckedResult = riskLevelSettings.lastChangeCheckedRiskLevelTimestamp
+        if (lastCheckedResult == newResult.calculatedAt) {
+            Timber.d("We already checked this risk level change, skipping further checks.")
+            return
+        }
+        riskLevelSettings.lastChangeCheckedRiskLevelTimestamp = newResult.calculatedAt
+
```

Comment:
is there any advantage to create the variable _lastCheckedResult_ ? can we not directly compare _riskLevelSettings.lastChangeCheckedRiskLevelTimestamp_ in the condition?

The comment is more for an information seeking purpose rather that a suggestion.

```suggestion
        if (iskLevelSettings.lastChangeCheckedRiskLevelTimestamp == newResult.calculatedAt) {
            Timber.d("We already checked this risk level change, skipping further checks.")
            return
        }
        riskLevelSettings.lastChangeCheckedRiskLevelTimestamp = newResult.calculatedAt

```

Reply:
Readability, inlining the variable exceeds our maximum line length.